src/main.ts

```diff
@@ -1,72 +1,417 @@
+ *
+ * @deprecated Use {@link ArmClient} instead.
+ * @alpha
+ * @group Raw Protobufs
+ */
+export { default as armApi } from './gen/component/arm/v1/arm_pb';
 export { type Arm, ArmClient } from './components/arm';
+
+/**
+ * Raw Protobuf interfaces for a Base component.
+ *
+ * Generated with https://github.com/improbable-eng/grpc-web
+ *
+ * @deprecated Use {@link BaseClient} instead.
+ * @alpha
+ * @group Raw Protobufs
+ */
+export { default as baseApi } from './gen/component/base/v1/base_pb';
 export { type Base, BaseClient } from './components/base';
+
+/**
+ * Raw Protobuf interfaces for a Board component.
+ *
+ * Generated with https://github.com/improbable-eng/grpc-web
+ *
+ * @deprecated Use {@link BoardClient} instead.
+ * @alpha
+ * @group Raw Protobufs
+ */
+export { default as boardApi } from './gen/component/board/v1/board_pb';
 export { type Board, BoardClient } from './components/board';
+
+/**
+ * Raw Protobuf interfaces for a Camera component.
+ *
+ * Generated with https://github.com/improbable-eng/grpc-web
+ *
+ * @deprecated Use {@link CameraClient} instead.
+ * @alpha
+ * @group Raw Protobufs
+ */
+export { default as cameraApi } from './gen/component/camera/v1/camera_pb';
 export { type Camera, CameraClient } from './components/camera';
+
+/**
+ * Raw Protobuf interfaces for a Motor component.
+ *
+ * Generated with https://github.com/improbable-eng/grpc-web
+ *
+ * @deprecated Use {@link MotorClient} instead.
+ * @alpha
+ * @group Raw Protobufs
+ */
+export { default as motorApi } from './gen/component/motor/v1/motor_pb';
 export { type Motor, MotorClient } from './components/motor';
+
+/**
+ * Raw Protobuf interfaces for a MovementSensor component.
+ *
+ * Generated with https://github.com/improbable-eng/grpc-web
+ *
+ * @deprecated Use {@link MovementSensorClient} instead.
+ * @alpha
+ * @group Raw Protobufs
+ */
+export { default as movementSensorApi } from './gen/component/movementsensor/v1/movementsensor_pb';
 export {
   type MovementSensor,
   MovementSensorClient,
 } from './components/movementsensor';
+
+/**
+ * Raw Protobuf interfaces generated with
+ * https://github.com/improbable-eng/grpc-web for a Sensor component.
+ *
+ * @deprecated Use {@link SensorClient} instead.
+ * @alpha
+ * @group Raw Protobufs
+ */
+export { default as sensorApi } from './gen/component/sensor/v1/sensor_pb';
 export { type Sensor, SensorClient } from './components/sensor';
+
+/**
+ * Raw Protobuf interfaces for a Sensors service.
+ *
+ * Generated with https://github.com/improbable-eng/grpc-web
+ *
+ * @example
+ *
+ * ```ts
+ * import { grpc } from '@improbable-eng/grpc-web';
+ *
+ * const client = {}; // replace with a connected robot client
+ *
+ * const request = new sensorsApi.GetSensorsRequest();
+ * request.setName('my sensors');
+ *
+ * client.sensorsService.getSensors(
+ *   request,
+ *   new grpc.Metadata(),
+ *   (error, response) => {
+ *     // do something with error or response
+ *   }
+ * );
```

Comment:
I'm only including this kind of example for components/services that do not have a wrapper yet - we should deprecate and remove the example once the corresponding wrapper is available.